configs/ruby/GPU_VIPER.py

```diff
@@ -338,6 +338,7 @@ def create(self, options, ruby_system, system):
         self.replacement_policy = TreePLRURP()
 
 
+# unused in GPU_VIPER; see git blame for discussion
```

Comment:
Shall I remove this then? @abmerop

Reply:
I don't have a strong preference, and it seems @mattsinc is strongly against removal.  I think adding the comment (what you have now) is a good compromise.  Will give Matt one more opportunity to review but it looks fine to me

Reply:
Cool, then it's ready to merge, I think

Reply:
Agreed, I will let Matt comment one more time (likely tomorrow).  In the meantime it looks like one of the commits are missing a Change-ID.  I'd also recommend squashing the commits as it looks like one is essentially undoing the other. Thanks,

Reply:
If others don't think it hurts their protocols and think this is fine, then we should just merge it -- once the issue @abmerop raised is fixed.

Reply:
Added the change ID; not sure if the squash is something I should do -- seems like one of the maintainers should? Otherwise please let me know.

Reply:
@mysoreanoop, you should squash the commits. 

https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History